docs/install.md

```diff
@@ -1,77 +1,224 @@
+        for details
+    *   `Compute Instance Admin`: Used to list instances and to attach disks to
+        instances
+    *   `Service Account User`: Used when attaching disks
+    *   `Cloud Functions Developer`: Used by turbiniactl to query task status
+*   Create a new key for your service account, and then point to it with an
+    environment variable:
+    *   `export
+        GOOGLE_APPLICATION_CREDENTIALS="/home/turbinia/turbinia-service-account-creds.json"`
+*   Add the service account to the gcloud auth **(RECOMMENDED)**
+    *   `gcloud auth list`
+    *   `gcloud auth activate-service-account
+        --key-file=$GOOGLE_APPLICATION_CREDENTIALS`
+*   Alternately you can run Turbinia under your own credentials **(NOT
+    RECOMMENDED)**
+    *   Run `gcloud auth login` (may require you to copy/paste url to browser)
+    *   Or run `gcloud auth application-default login`
+*   Continue to [Inside the Virtualenv](#3-inside-the-virtualenv)
+
+### 3. Inside the Virtualenv
+
+#### Create a Virtualenv Instance
+*   Create (once) and activate Virtualenv
+    *   `virtualenv turbinia-env && . turbinia-env/bin/activate`
+    *   ***NOTE:*** The next time you need to use virtualenv, just log in as
+        turbinia and activiate virtualenv without recreating it
+*   Do not exit the Virtualenv until you have completed all the steps!
+
+#### Build and Configure
+
+*   Install Turbinia
+    *   `git clone https://github.com/google/turbinia.git`
+    *   `pip install --upgrade pip`
+    *   `pip install -r turbinia/requirements.txt`
+*   Install Plaso
+    *   `curl -o plaso-requirements.txt
+        https://raw.githubusercontent.com/log2timeline/plaso/master/requirements.txt`
+    *   `pip install -r plaso-requirements.txt`
+*   Update Python Path
+    *   Until [Issue #39](https://github.com/google/turbinia/issues/39) is
+        fixed, we need to adjust the `$PYTHONPATH` from the root of the Turbinia
+        repository:
+        *   ``cd turbinia; export PYTHONPATH=$PYTHONPATH:`pwd`; cd -``
+    *   And set it in your .bashrc file:
+        *   ``cd turbinia; echo "export PYTHONPATH=$PYTHONPATH" >> ~/.bashrc ; cd -``
+*   Choose one of the locations for storing the Turbinia config and configure
+    from there:
+    *   `/etc/turbinia/turbinia.conf` **(RECOMMENDED)**
+        *   `sudo mkdir /etc/turbinia`
+        *   `cp <localgitpath>/turbinia/config/turbinia_config.py /etc/turbinia/`
+    *   `/home/turbinia/.turbinia`
+        *   `cp <localgitpath>/turbinia/config/turbinia_config.py /home/turbinia/.turbiniarc`
+    *   Directly configure `<localgitpath>/turbinia/config/turbinia_config.py`
+    *   ***NOTE***: Match the `PUBSUB_TOPIC` variable in the configuration to the name
+        of the topic and subscription you created in the GCP.
+*   Continue to [Deploy the Cloud Functions](#deploy-the-cloud-functions)
+
+#### Deploy the Cloud Functions
+
+*   `cd <localgitpath>/turbinia/tools/gcf_init && ./deploy_gcf.py`
```

Comment:
I tested the installation instruction by following it exactly.

I didn't realize deploy_gcf.py ran gcloud functions deploy with --stage-bucket, which requires GCS. Also, it requires all the configuration varables to be set prior to moving forward. Another thing is if the config needs to be fully set up, I will have to reorder the doc a bit so that GCS is created first before making changes to the config file.

Reply:
The GCS instruction is already included. So, we are good here.